### ISSUE TITLE
Generator for Rust

### ISSUE THREAD

**Issue by calavera** [chronoxor/FastBinaryEncoding#18]
Hi!

This project looks very promising. Thanks for providing those benchmarks, they are super interesting.

I have not seen any mentions in the issues, so I thought I'd open an issue to see if there is any interest in creating a generator for Rust.

**Comment by dendisuhubdy** [comment]
Yes, was just about to raise this issue for a support. Is this open for a discussion? Could we do a PR?

**Comment by chronoxor** [comment]
Rust generator is not in near plans. But PR with the generator is very welcome!
